tests/simulate.py

```diff
@@ -24,7 +24,7 @@
 from astropy.time import Time
 import astropy.io.ascii
 
-# RASCIL 0.2.0b0 imports nifty_gridder but doesn't install it. It's not needed
+# RASCIL 0.2.0 imports nifty_gridder but doesn't install it. It's not needed
```

Comment:
Actually the release notes for RASCIL 0.2.0 [here](https://gitlab.com/ska-telescope/external/rascil/-/tags/v.0.2.0) say nifty_gridder is automatically installed and it is in 0.2.0's [requirements.txt](https://gitlab.com/ska-telescope/external/rascil/-/blob/v.0.2.0/requirements.txt#L25).

I guess it still isn't needed here though?

Reply:
It doesn't seem to be required by setup.py though: if I just pip install rascil then it doesn't install nifty gridder (it looks like there have since been some changes to setup.py in master). But I can add nifty-gridder to the requirements file rather than working around it.

Reply:
They seem to have explicitly added it to setup.py in master - presumabley to work around whatever issue stopped setup.py from using it. So I imagine 0.2.1 will have it 😄

Reply:
Done.